R/ResultFlagsIndependent.R

```diff
@@ -265,6 +265,8 @@ TADA_FlagContinuousData <- function(.data, clean = FALSE, flaggedonly = FALSE, t
 
   if (length(noncont.data) >= 1) {
     info_match <- noncont.data %>%
+      # remove quality control samples
+      dplyr::filter(!grepl("Quality Control", ActivityTypeCode)) %>%
```

Comment:
There are a few ActivityTypeCode's that do not include "Quality Control" that are also QC samples. Could you reference the internal TADA package WQXActivityTypeRef here instead and keep only the results assigned by TADA as "Non_QC"?

Reply:
Yes - that is a better, more specific solution. I can make that change.